policyengine_us/parameters/gov/states/mt/tax/income/credits/elderly_renter_credit/cap.yaml

```diff
@@ -0,0 +1,16 @@
+scription: Montana used cap in credit computation.
```

Comment:
```suggestion
scription: Montana provides this elderly renter credit cap.
```

Reply:
use same format in all yaml file descriptions. "Montana does xyz for the elderly renter credit."